### DISCUSSION TITLE
Is there a better way to support vue now?

### DISCUSSION THREAD

**Discussion by skywalker512** [web-infra-dev/rspack#2106] [Q&A]
rspack is great, it can reduce build time from 50s to 3s in my personal project.

I created a modified version of vue loader. 

https://github.com/skywalker512/vue-rsloader

Since I am not very familiar with webpack, I would like to ask if there are better solutions now.

**Comment by skywalker512** [comment]
I also want to ask a question, that is, there is a source map problem with the [modified vue loader](https://github.com/skywalker512/vue-rsloader) and your [official example](https://github.com/web-infra-dev/rspack/tree/main/examples/vue). But it's normal in webpack.

## [official example](https://github.com/web-infra-dev/rspack/tree/main/examples/vue)
<img width="839" alt="image" src="https://user-images.githubusercontent.com/12960906/224208741-4aa93b82-6ee7-4033-8b3f-6bf8fc6e44b6.png">

## [modified vue loader](https://github.com/skywalker512/vue-rsloader)
<img width="1880" alt="image" src="https://user-images.githubusercontent.com/12960906/224300650-b696af98-7165-4264-9114-9dc2d5137809.png">

**Reply by hardfist** [reply]
our modified vue-loader is just a poc we're going to support official vue-loader in the future for better vue support
